app/assets/stylesheets/dm/pages/_practice.scss

```diff
@@ -478,6 +478,26 @@
   }
 }
 
+.remove-main-display-image-link {
+  margin-top: 6px;
+  margin-bottom: 24px;
+  color: $dm-color-high-red;
+  text-decoration: underline;
+
+  &:hover {
+    color: #b51d09; /* TODO: check if this is needed */
```

Comment:
⚠️ Saw these hover styles for other links so kept this for the remove and upload main display image as well